scripts/qa_crowbarsetup.sh

```diff
@@ -152,6 +152,8 @@ function mount_localreposdir_target
     fi
     mkdir -p $localreposdir_target
     if ! grep -q "$localreposdir_target\s\+$localreposdir_target" /etc/fstab ; then
+        echo 'force_drivers+="9pnet_virtio"' > /etc/dracut.conf.d/03-libvirt.conf
+        dracut -f
```

Comment:
well, this is running only once, as it's "guarded" by the contents of `/etc/fstab` ?